views/add-new-job-roles.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Add Job Role</title>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+        <link href="/public/styles.css" rel="stylesheet">
+        <link rel="icon" type="image/x-icon" href="https://www.kainos.com/favicon.png">
+    </head>
+    <body>
+        <header>
+            {% include "header.html" %}
+        </header>
+        <main class="container">
+
+            <h1>Add Job Role</h1>
+
+            <section id = "form">
+                <div class="container py-3">
+                    <div id="error-container">
+                        {% if errormessage %}
+                        <div id="error-container">
+                          <div class="alert alert-danger">
+                            {% errormessage %}
```

Comment:
My bad. This should be {{ errormessage }}
```suggestion
                            {{ errormessage }}
```